ui/webpack.prod.js

```diff
@@ -0,0 +1,21 @@
+const merge = require('webpack-merge');
+const webpack = require('webpack');
+
+const common = require('./webpack.common.js');
+
+const LANGS = ['en', 'es'];
+
+module.exports = LANGS.map(function(lang) {
```

Comment:
May be good to be consistent whether to use `function` or `=>` for anonymous functions. I guess `=>` is preferred recently?

As a side question, are you following any style guide for JS code? I believe the existing JS code in PF doesn't really follow any style guides...

Reply:
Changed it to =>.
Re: the style guide: I think we could mostly follow Google's JS style guide:
https://google.github.io/styleguide/jsguide.html
the problem is that it seems to assume you're using Closure (there's a bunch of rules about where to put goog.module and stuff like that), and we're not using Closure, but we could just ignore those parts.

Reply:
Sounds good. Maybe we should add pointer to the style guide here, which has a pointer to Python Style Guide?:
https://github.com/google/personfinder/blob/master/CONTRIBUTING.md#for-code-reviewers

Reply:
Done.